script.js

```diff
@@ -0,0 +1,7 @@
+const hamburger = document.querySelector('.hamburger ');
+const navMenu = document.querySelector('.menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
```

Comment:
- [ ] Kindly make sure when one of the navigation links is clicked, the menu is closed automatically and the user will be redirected to the part of the page according to which was clicked ( FOR EXAMPLE: if the `About` button is clicked, the website will navigate to the `About myself` section).

   - ### YOUR APP BEHAVIOUR

   ![chrome-capture-2023-2-15](https://user-images.githubusercontent.com/87197412/225392068-73c1cc6b-ad82-4b80-b1da-cf976a1cee99.gif)

Reply:
- [ ] You did a great job following the previous reviewer's advice on displaying the appropriate section of your page! 🚀 However, when I scroll back to the top, the menu is still open:

[Screencast from 15-03-23 16:33:20.webm](https://user-images.githubusercontent.com/77069011/225449827-55e48475-84a9-479c-b40a-725d6d523153.webm)

Please consider fixing this behavior! According to the requirements: 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/225449992-d5567a54-658b-42b7-ab5e-234f6ddfc515.png)